Makefile

```diff
@@ -71,6 +72,23 @@ endif
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Image building tool (docker / podman)
+# Image building tool (docker / podman) - docker is preferred in CI
+OCI_BIN_PATH := $(shell which docker 2>/dev/null || which podman)
+OCI_BIN ?= $(shell basename ${OCI_BIN_PATH})
+
+# build a single arch target provided as argument
+define build_target
+	echo 'building $(1) for arch $(2)'; \
+	DOCKER_BUILDKIT=1 $(OCI_BIN) buildx build --load --build-arg TARGETPLATFORM=linux/$(2) --build-arg TARGETARCH=$(2) --build-arg BUILDPLATFORM=linux/amd64 -t  $(1)-$(2) -f $(3) .;
```

Comment:
Split across multiple lines. I think there are a few other lines that are really long that might need to be split up.

Reply:
done